apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.py

```diff
@@ -401,7 +402,7 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
         "allow-blank",
         "size",
         "placeholder",
-        "label",
+        "aria-label",
```

Comment:
Same as other comment, this is a backwards incompatible change.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/11857#discussion_r2067293369